<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Studie</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="German" country="Germany"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>746</width>
      <height>591</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="German" country="Germany"/>
    </property>
    <widget class="QWidget" name="introduction">
     <widget class="QLabel" name="intro_text_1">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>10</y>
        <width>751</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Danke, dass du an dieser Studie teilnimmst! &lt;/p&gt;&lt;p&gt;Deine Aufgabe ist es, die angezeigten Texte so &lt;span style=&quot; font-weight:600;&quot;&gt;schnell und fehlerfrei &lt;/span&gt;wie möglich einzugeben. Solltest du mal einen &lt;/p&gt;&lt;p&gt;Fehler machen, ignoriere ihn und tippe einfach weiter. &lt;span style=&quot; font-weight:600;&quot;&gt;Du brauchst Fehler nicht zu korrigieren!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Sobald du auf den &amp;quot;Experiment starten&amp;quot; - Button unten klickst, beginnt ein kurzer Testdurchlauf, bevor du mit der &lt;/p&gt;&lt;p&gt;richtigen Aufgabe startest. Diese wird im Anschluss 3 mal wiederholt. Die genaue Aufgabenstellung variiert &lt;/p&gt;&lt;p&gt;dabei zwischen normaler Texteingabe und Texteingabe mit Autovervollständigung. Am Ende musst du noch &lt;/p&gt;&lt;p&gt;einen kurzen Fragebogen ausfüllen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_study_btn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>450</y>
        <width>141</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 207, 0);</string>
      </property>
      <property name="text">
       <string>Experiment starten</string>
      </property>
     </widget>
     <widget class="QLabel" name="intro_text_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>751</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bitte achte darauf, während der Dauer dieses Experiments ungestört und konzentriert zu bleiben.&lt;/p&gt;&lt;p&gt;Sobald du bereit bist, drücke bitte auf den &amp;quot;Experiment starten&amp;quot;-Button.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="example_trial">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>258</width>
            <height>68</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="example_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Aufgabenstellung:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>258</width>
            <height>68</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="example_task_description">
        <property name="text">
         <string>Lorem Ipsum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Unten siehst du ein Beispiel, das du im Textfeld unten testen kannst. Sobald du bereit bist, klick auf den Button unten, um mit der richtigen Aufgabe zu beginnen.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>39</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="example_text_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; text-decoration: underline;&quot;&gt;Beispieltext:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="example_text">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Dolor sit amet</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>39</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="example_text_box_container"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>318</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="start_actual_study_btn">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 207, 0);</string>
          </property>
          <property name="text">
           <string>Weiter</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>318</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="text_entry_task">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="entry_task_description">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bitte gib den untenstehenden Text &lt;span style=&quot; font-weight:600;&quot;&gt;so schnell und fehlerfrei wie möglich&lt;/span&gt; in das Textfeld ein (Copy &amp;amp; Paste ist selbsverständlich nicht erlaubt). &lt;span style=&quot; font-weight:600;&quot;&gt;Fehler müssen dabei nicht korrigiert werden!&lt;/span&gt; Sobald du fertig bist, klicke auf den Button unten.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>53</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="task_text_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	 border: 1px solid black;
}</string>
        </property>
        <property name="text">
         <string>Lorem Ipsum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>52</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="task_text_box_container"/>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>53</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>308</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="task_finished_btn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 207, 0);</string>
          </property>
          <property name="text">
           <string>Weiter</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>308</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="post_questionnaire">
     <widget class="QSlider" name="keyboard_affinity_slider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>511</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="gender_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Geschlecht:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="age_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dein Alter:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="send_questionnaire_btn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>530</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 207, 0);</string>
      </property>
      <property name="text">
       <string>Senden</string>
      </property>
     </widget>
     <widget class="QLabel" name="keyboard_affinity_high">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>330</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>sehr häufig</string>
      </property>
     </widget>
     <widget class="QComboBox" name="gender_selection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>87</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Männlich</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Weiblich</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Divers</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="keyboard_affinitiy_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>511</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bitte gib an, wie häufig du im Alltag eine Tastatur verwendest:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="keyboard_affinity_low">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>330</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>sehr selten</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="age_selection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>52</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimum">
       <number>18</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="occupation_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Beruf :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="occupation_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>521</width>
        <height>32</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="questionnaire_intro">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bitte nimm dir noch kurz Zeit, um den folgenden Fragebogen auszufüllen:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QSlider" name="speed_estimation_slider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>511</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="speed_estimation_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>511</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Wie würdest du deine Tippgeschwindigkeit einschätzen?&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="speed_estimation_slow">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>440</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>sehr langsam</string>
      </property>
     </widget>
     <widget class="QLabel" name="speed_estimation_fast">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>440</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>sehr schnell</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="finish_page">
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>181</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(212, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Studie beenden</string>
      </property>
     </widget>
     <widget class="QLabel" name="thank_you_text">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vielen Dank für die Teilnahme!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_trial_btn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>141</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 211, 64);</string>
      </property>
      <property name="text">
       <string>Nächster Durchgang</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
